ext/data/schemas/dictionary-index-schema.json

```diff
@@ -42,9 +42,22 @@
             "type": "string",
             "description": "Creator of the dictionary."
         },
+        "isUpdatable": {
+            "type": "boolean",
+            "const": true,
+            "description": "Whether this dictionary contains links to its latest version."
+        },
+        "indexUrl": {
+            "type": "string",
+            "description": "URL for the index file of the latest revision of the dictionary, used to check for updates."
+        },
+        "downloadUrl": {
```

Comment:
Might be a dumb question but does this expose yomitan dictionaries to be a vector for malware?

Reply:
I'm counting on the extension APIs not letting us do anything stupid. I guess the download from URL and unzip happens in memory so a zip bomb or smth like that isn't possible?

Reply:
Zip bombs rely on recursive unzipping. You can unzip a single layer of a zip bomb and it wont create any problem.

I don't think it's possible for malware to do anything funny in the json since we nail stuff down with the schema validations. The only thing would be structured content possibly doing something weird but we dont let JS run in there so it would have to be a very major browser bug to get some sort of malware into pure html.